### PR TITLE
Update sssp_project.md

### PR DESCRIPTION
**Pull request by miwakotsuji**
The report with the results from 2017-2018

### CODE REVIEW

File: _projects/sssp_project.md [hunk 1]
```diff
@@ -41,14 +41,18 @@ We consider a machine learning methodology to give appropriate weighting factors
...
+
 ## Visits and meetings
 
 Miwako Tsuji visited NCSA in January 2017.
 
 ## Impact and publications
 
-None yet.
-
+Miwako Tsuji, William T. C. Kramer and Mitsuhisa Sato,"A Performance Projection of Mini-Applications onto Benchmarks Toward the Performance Projection of Real-Applications", 2017 IEEE International Conference on Cluster Computing (CLUSTER) Workshop on Representative Applications (WRAp)
```
Comment (pancetta) [inline _projects/sssp_project.md:56]: Please use the bib files for citing/referencing, in particular for JLESC papers. See [jlesc.bib](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) for the bib file and e.g. [this project](https://jlesc.github.io/projects/ichpc_project/) for an example.

File: _projects/sssp_project.md [hunk 2]
```diff
@@ -79,7 +83,6 @@ Remember to use the `--file jlesc.bib` with the `cite` tag.
 
 ## Future plans
 
-See above.
```
Comment (pancetta) [inline _projects/sssp_project.md:86]: Are there any future plans? Is the project finished? If so, please change the status. If not, please add a line here.

**Review by pancetta — CHANGES_REQUESTED** [review]

File: _projects/sssp_project.md (hunk 2 above)
Comment (miwakotsuji) [reply _projects/sssp_project.md:86]: I've added the plan.

Comment (pancetta) [reply _projects/sssp_project.md:86]: Still need to submit, I can't see it online. The same goes for the reference fix in the text above.

**Pull request closed without merging**